service-manual/user-centred-design/resources/patterns/check-your-answers-pages.md

```diff
@@ -37,7 +37,7 @@ to drop out of the transaction before completing it
 
 # Code
```

Comment:
Then it would have no heading? I suggest we merge this fix to a published broken link and then discuss other changes